docs/snippets/ov_preprocessing_migration.py

```diff
@@ -1,102 +0,0 @@
-# Copyright (C) 2018-2023 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-#
-
-from utils import get_model, get_ngraph_model
-
-#! [ov_imports]
-from openvino import Core, Layout, Type
-from openvino.preprocess import ColorFormat, PrePostProcessor, ResizeAlgorithm
-#! [ov_imports]
```

Comment:
Here you removed also API 2.0 imports and code. Please revert it.

Reply:
this is migration guide.
We have two another snippets which show how to use preprocessing and properties.